recipe/meta.yaml

```diff
@@ -44,7 +51,7 @@ requirements:
     - filelock
     - pytorch =*=cuda*
     - lit
-    - cudatoolkit
+    - cuda-version >={{ cuda_compiler_version }}  # [cuda_major >= 12]
```

Comment:
For CUDA 12, `{{ compiler('cuda') }}` already adds [`cuda-version` via `run_exports`]( https://github.com/conda-forge/cuda-nvcc-feedstock/blob/394f3817decd9316269b5a40e5076b8bcc94dc33/recipe/meta.yaml#L43-L45 )

Also for CUDA 11, `{{ compiler('cuda') }}` adds [`cudatoolkit` via `run_exports`]( https://github.com/conda-forge/nvcc-feedstock/blob/c01599568a71b44a3026970d6b5666f9c7e0e8be/recipe/meta.yaml#L40-L46 )

So think we can drop this too (as well as the `cudatoolkit` that was here before)

```suggestion
```